users/user-manager.js

```diff
@@ -0,0 +1,74 @@
+var response = require('../Objects/response.js');
+var userModels = require('./user-models.js');
+var Student = userModels.Student;
+var Admin = userModels.Admin;
+
+var createUser = (reqData, createCallback) => {
+  if (reqData.role == 'Student') {
+    // Create new student.
+    var student = new Student({
+      approvalStatus: false,
+      email: reqData.email,
+      firstName: reqData.firstName,
+      lastName: reqData.lastName,
+      points: 0,
+      role: 'Student',
+      studentId: 0
```

Comment:
I meant to add this comment to another thread (in which he mentioned it's the UID).  I've added this comment to the correct comment thread.